playwright/inscription-test-project/dataclasses/ch/ivyteam/test/Person.d.json

```diff
@@ -2,6 +2,7 @@
   "$schema" : "https://json-schema.axonivy.com/data-class/12.0.0/data-class.json",
   "simpleName" : "Person",
   "namespace" : "ch.ivyteam.test",
+  "comment" : "A registrated webshop user",

```

Comment:
tried to show this in the help; but it seems like we don't send the 'description comment' as help for dataclasses :shrug: